src/osd/ECBackend.cc

```diff
@@ -728,8 +728,8 @@ int ECBackend::recover_object(
   }
   h->ops.back().recovery_progress.omap_complete = true;
   for (set<pg_shard_t>::const_iterator i =
-	 get_parent()->get_actingbackfill_shards().begin();
-       i != get_parent()->get_actingbackfill_shards().end();
+	 get_parent()->get_acting_recovery_backfill_shards().begin();
+       i != get_parent()->get_acting_recovery_backfill_shards().end();
```

Comment:
I think the repeating call `get_parent()->get_acting_recovery_backfill_shards()` is inefficiency.
Maybe
```c++
const set<pg_shard_t> acting_recovery_backfill_shards = get_parent()->get_acting_recovery_backfill_shards();
for (set<pg_shard_t>::const_iterator i = acting_recovery_backfill_shards.begin();
       i != acting_recovery_backfill_shards.end(); ++i)
```
is better. What do you think？